crates/sui-graphql-rpc/schema/draft_schema.graphql

```diff
@@ -208,6 +208,12 @@ input ObjectFilter {
   owner: SuiAddress
   objectIds: [SuiAddress!]
   objectKeys: [ObjectKey!]
```

Comment:
it's not about this PR and sry if I missed the discussion, `ObjectKey` includes both object ID and version, why do we still need `objectIds`, can we make `version` in `ObjectKey` instead?

Reply:
got it, with compound queries we will need to handle the cases of `object_ids_query AND / OR object_keys_query` on the server end, although this would not be a common legit use-case; devs will need to learn 2 flavors instead of one, too.
I am not strongly opinionated but prefer the simplicity on both interface and impl over the extra convenience here.

Reply:
This is a purely syntactic convenience -- as a first step, the GraphQL data loader would translate a query by ObjectIDs into one on ObjectKeys, but with no version, so there is really no additional complexity here and this feature was requested by @hayes-mysten from front-end & apps, who I would tend to defer to on what is more convenient to offer -- I haven't used GraphQL APIs in anger enough to have a strong opinion!